<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomViewer</class>
 <widget class="QMainWindow" name="DicomViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DicomViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="FileOpenButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件1</string>
    </property>
   </widget>
   <widget class="QLabel" name="DicomShow1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FileOpenButton2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件2</string>
    </property>
   </widget>
   <widget class="QLabel" name="DicomShow2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setLungWin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置肺窗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setMedisatWin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置纵膈窗</string>
    </property>
   </widget>
   <widget class="QLabel" name="cur_WinWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前窗宽：0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cur_WinCen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前窗位：0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="exitButton">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
